shared/base/src/main/java/com/ivy/base/model/LoanRecordType.kt

```diff
@@ -16,4 +16,6 @@ fun <T> LoanRecordType.processByType(decreaseAction: () -> T, increaseAction: ()
         LoanRecordType.DECREASE -> decreaseAction()
         LoanRecordType.INCREASE -> increaseAction()
     }
-}
\ No newline at end of file
+}
+
+fun defaultLoanRecordType(): LoanRecordType = LoanRecordType.DECREASE
```

Comment:
Let's directly use `LoanRecordType.DECREASE` instead of this function cuz it's simpler, shorter and more straightforward to understand

Reply:
@Vishwa-Raghavendra ping me when you've resolved it so we can merge

Reply:
Hey @ILIYANGERMANOV 
Update the code with requested changes,